src/srr-mapping.ts

```diff
@@ -695,8 +719,10 @@ export function handleMigrateSRR(event: MigrateSRREvent): void {
   logInvocation("handleMigrateSRR", event);
   let srrId = event.params.tokenId.toString();
   let srr = SRR.load(srrId);
-  srr.originChain = event.params.originChain;
-  srr.save();
+  if (srr) {
```

Comment:
little worried about changing this ...  these events were one time only and we know the previous logic was working fine. i think we can safely assume srr exists because if migrateSRR was emitted it was for an already existing srr too.

Reply:
I see, I try to revert this line.

Reply:
I put this line in because I get a compile error in the IDE. However, build is fine, so I put this line back in.